experimental/generation/generator/packages/library/templates/generator.lg

```diff
@@ -155,7 +155,7 @@
 
 # actionAssign(entity)
 > Assign entity to current property
-- ${list(where([actionSetOrPush(`@${entity}`), actionDeletePropertyToChange()], actions, actions != ''))}
+- ${list(where([actionSetOrPush(`${entity}`), actionDeletePropertyToChange()], actions, actions != ''))}
 
```

Comment:
This tells me you have not tried to actually tested a generated dialog.  I always build sandwich and interact with it to make sure scenarios work.  Removing @ is only for labels in LU files.  Here this is how you refer to an entity and it should be here.

Reply:
You mean the whole generator.lg should not be changed?